week1/project/hackyourweather/src/App.js

```diff
@@ -0,0 +1,14 @@
+import logo from './logo.svg';
+import './App.css';
+import MainWeather from './main';
```

Comment:
In 99% of the time you call you file the same name as your component. I would just call this `WeatherList` it's more descriptive. 

```suggestion
import MainWeather from './MainWeather';
```